### DISCUSSION TITLE
Enhancement for the FindAll windows

### DISCUSSION THREAD

**Discussion by yoann-darche** [zodiacon/TotalRegistry#38] [Ideas]
It would be a good enhancement to put the findAll windows in non-modal mode and have the possibility to synchronize a result path (found with the FindAll dialog) to the main windows in order to make some tasks (like edit/extract/delete).

For now, each time you perform a search with the findall, you cannot navigate thru the result easily, each time you select a result, you lose your search, and need to redo it. I have not understand the utility of the Save/Load button!

thanks !

**Comment by zodiacon** [comment]
Thanks, I will consider making the search dialog modeless.
The Save/Load results is supposed to save the results to a file... not implemented yet :(

**Comment by zodiacon** [comment]
Made Find all dialog modeless.
